scitt-components/scitt-rbac.md

```diff
@@ -0,0 +1,36 @@
+# SCITT Role Based Access Control
+
+> TODO:
+> - Why we need limited access to information submitted to SCITT 
+> - Will SCITT support partitioned access within a SCITT instance, or will a SCITT instance be limited and only approved content is promoted to another SCITT instance that has broader access?
+
+## Promotion Across RBAC Instances
+
+Whether a SCITT instance supports RBAC partitioning or not, users will configure multiple SCITT instances to support their dev through production workflows, or their internal systems being maintained separate from their public services. Users can promote the content from internal, secured SCITT instances, to SCITT instances with different permissions, or public access.
```

Comment:
I like RBAC / ABAC.... but I think this is the wrong layer to apply them.

There will be traditional databases that feed off scitt instances, that have out of the box / off the shelf solutions... We should leave complexity to those layers.

The SCITT Instance should have a very simple `write` and `read` permission, with logging.

Expectation should be that its append only and available to forensic teams, but they are never going to query over it directly... instead they will dump its content to a tool that supports their mission.

Reply:
I'm not objecting to this discussion in the use cases though, just concerned about the surface areas for a SCITT instance.

Reply:
For the indexing, this is another great topic. The SCITT Ledger will be quite full, and do we want to encourage other various services to continually pull everything to then enable users to query external systems that are never up to date?
Or, do we provide hooks in the SCITT apis to index real-time, with no duplicate of data? 
There's a page for scitt indexing that we riff on the design options.

Reply:
I'm going to resolve this one, for now, and let this particular doc evolve with additional PRs and comments.